File: 2013-06-07/python/exercise1.py
from pyplasm import *
import random

dom1D = INTERVALS(60)(16)
dominio = PROD([dom1D,dom1D])
DRAW = VIEW

def montagne(punto):
	x = punto[0];
	y = punto[1];
	z =  (random.random()*3 * SIN(x) * COS(y))**2;
	ran = random.random()/5;
	return [x, y, ran+z];

territorio = MAP(montagne)(dominio);
territorio = T([1,2])([-30,-30])(territorio)
DRAW(territorio);

File: 2013-06-07/javascript/exercise1.js
var dominio = DOMAIN([[-30, 30], [-30, 30]])([30,30]);

var montagneF = function(punto){
	var x = punto[0];
	var y = punto[1];
	//var z =  Math.random()*3 * SIN(x) * COS(y);
	var z =  Math.pow(Math.random()*3 * SIN(x) * COS(y),1.3);
	var random = Math.random()/5;
	z = random + z;
	return [x, y, z];
}
var territorio = MAP(montagneF)(dominio);

DRAW(COLOR([160/255, 82/255, 45/255])(territorio));